api/modules/files/model.js

```diff
@@ -0,0 +1,22 @@
+const BaseModel = require('../../classes/base_model');
+
+const instanceProps = {
+  tableName: 'files',
+  comic: function () {
+    return this.belongsTo(require('../projects/model'));
+  }
+};
+
+const classProps = {
+  typeName: 'files',
+  filters: {
+    project_id: function (qb, value) {
+      return qb.whereIn('project_id', value);
+    }
+  },
+  relations: [
+    'project'
+  ]
+};
+
+module.exports = BaseModel.extend(instanceProps, classProps);
```

Comment:
Now that I've taken a closer look at this, I really like the way these relations are declared. Is this also what defines them in the db itself?